profile/profile.go

```diff
@@ -674,6 +674,29 @@ func numLabelsToString(numLabels map[string][]int64, numUnits map[string][]strin
 	return strings.Join(ls, " ")
 }
 
+// SetTag sets the specified key to the specified value for all samples in the
+// profile.
+func (p *Profile) SetTag(key string, value []string) {
+	for _, sample := range p.Sample {
+		if sample.Label == nil {
+			sample.Label = make(map[string][]string)
```

Comment:
`sample.Label = map[string]string{}` is a bit cheaper.

Reply:
done.